### DISCUSSION TITLE
Install fails in Python 2.7

### DISCUSSION THREAD

**Discussion by dougbrowning** [python-pillow/Pillow#5626] [Q&A] [answered]
Trying to use easy_install-2.7 Pillow and it fails here
![image](https://user-images.githubusercontent.com/6051277/126572000-01d01fdc-440b-496f-b1f4-ad523fed0c11.png)

Thanks for any help.  

Just trying to edit Exif info and after like 4 diff ways none work.

**Comment by radarhere** [comment, marked as answer]
The last version to support Python 2.7 is Pillow 6.2.2 - https://pillow.readthedocs.io/en/stable/installation.html#python-support

Please either upgrade Python, or downgrade to Pillow 6.2.2.

**Reply by dougbrowning** [reply]
Ok great and how do I get Pillow 6.2.2.  

Thanks

**Reply by radarhere** [reply]
If your original command was `easy_install-2.7 Pillow`, then it should be `easy_install-2.7 Pillow==6.2.2`
